tests/build_feedstock_test.py

```diff
@@ -263,3 +263,44 @@ def test_build_feedstock_conda_pkg_format(mocker):
 
     pkg_format = "conda"
     opence._main(["build", build_feedstock.COMMAND, "--conda_pkg_format", pkg_format])
+
+def test_build_feedstock_default_debug(mocker):
+    '''
+    Tests that the default arguments for 'build_feedstock' generate the correct 'conda_build.api.debug' input args.
+    '''
+    mocker.patch(
+          'os.getcwd',
+          return_value="/test/test_recipe")
+    mocker.patch(
+          'os.path.exists',
+          return_value=False)
+    activation_str = '''
+      ################################################################################
+      Build and/or host environments created for debug. To enter a debugging environment:
+      cd /Users/UserName/miniconda3/conda-bld/debug_1542385789430/work && 
+      source /Users/UserName/miniconda3/conda-bld/debug_1542385789430/work/build_env_setup.sh
+      ################################################################################
+   '''
+
+    mocker.patch('conda_build.api.debug',
+                 side_effect=(lambda x,**kwargs:helpers.validate_conda_debug_args(x,expect_activation_string=activation_str)))
+    opence._main(["build", build_feedstock.COMMAND,"--debug"])
+
+def test_build_feedstock_default_debug_output_id(mocker):
+    '''
+    Tests that the default arguments for 'build_feedstock' generate the correct 'conda_build.api.debug --debug output id' input args.
+    '''
+    mocker.patch('os.path.exists',return_value=True)
+    recipe_path = os.path.join(test_dir, "test_recipe")
+    test_recipe_config = {'recipes' : [{'name' : 'debug_output_id', 'path' :recipe_path}]}
+    mocker.patch('open_ce.conda_utils.render_yaml', return_value=test_recipe_config)
+    activation_str = '''
+      ################################################################################
+      Build and/or host environments created for debug output id {}. To enter a debugging environment:
+      cd /Users/UserName/miniconda3/conda-bld/debug_1542385789430/work && 
+      source /Users/UserName/miniconda3/conda-bld/debug_1542385789430/work/build_env_setup.sh
+      ################################################################################
+   '''.format('output_1')
```

Comment:
`'output_1'` can be declared as a string variable and then it can be used at all the places.